prime-router/end-to-end-test.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+# Run a nice end to end test, covering all our formats, and all our transports,
+# and many different schemas.
+# 
+# This assumes a localhost prime/router is already running.
+# This assumes batch is on a 1-minute timer.
+#
+# This script is very ugly and repetitive.  Replace with kotlin.
+#
```

Comment:
If you have the local prime-router docker-compose up and and running, you should be able to just run this script out of the box.   You have to eyeball the results, but I tried to make them very simple to read.   I plan to write this in Kotlin to make it much more malleable.